webp-server.go

```diff
@@ -111,6 +112,9 @@ Develop by WebP Server team. https://github.com/webp-sh`, version)
 		ServerHeader:          "Webp Server Go",
 		DisableStartupMessage: true,
 	})
+	app.Use(etag.New(etag.Config{
+		Weak: true,
```

Comment:
Question: why use weak etag here?

Reply:
I assumed we wanted to keep it computationally cheap: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag#directives - Happy to use non-weak etags if you want